alerting/src/main/kotlin/org/opensearch/alerting/util/clusterMetricsMonitorHelpers/SupportedClusterMetricsSettingsExtensions.kt

```diff
@@ -39,9 +41,23 @@ import org.opensearch.commons.alerting.model.ClusterMetricsInput
 fun executeTransportAction(clusterMetricsInput: ClusterMetricsInput, client: Client): ActionResponse {
     val request = resolveToActionRequest(clusterMetricsInput)
     return when (clusterMetricsInput.clusterMetricType) {
+        ClusterMetricsInput.ClusterMetricType.CAT_INDICES -> {
+            request as CatIndicesRequestWrapper
+            val healthResponse = client.admin().cluster().health(request.clusterHealthRequest).get()
```

Comment:
`.get()` looks to be a blocking call. Should update to use suspendUntil instead.

Reply:
Thanks for the feedback!

Revised `executeTransportAction` to use `suspendUntil` instead of `get`. I reran the tests locally, and they continue to pass.